smart-contracts/contracts/mixins/MixinLockCore.sol

```diff
@@ -17,7 +18,8 @@ contract MixinLockCore is
   IERC721Enumerable,
   Ownable,
   MixinFunds,
-  MixinDisableAndDestroy
+  MixinDisableAndDestroy,
+  MixinLockManagerRole
```

Comment:
Could we also include this in the PublicLock inheritance order?  We have all of them listed there even though that's clearly not required - it helps to communicate the absolute ordering of contracts.

Reply:
yes, will do.